app/views/install/compose.phtml

```diff
@@ -355,52 +355,6 @@ services:
       - _APP_EXECUTOR_SECRET
       - _APP_USAGE_STATS
 
-  appwrite-executor:
```

Comment:
Why are we removing the executor here?

Reply:
There is a second executor higher up in the file: https://github.com/appwrite/appwrite/blob/1db9a872d9803047960aeb80e14b92c6d0f45201/app/views/install/compose.phtml#L159